sphinx/builders/epub3.py

```diff
@@ -157,30 +169,23 @@ def build_navlist(self, nodes):
                 continue
             if node['level'] > self.config.epub_tocdepth:
                 continue
-            if node['level'] == level:
-                navlist.append(self.new_navlist(node, level))
-            elif node['level'] == level + 1:
-                navstack.append(navlist)
-                navlist = []
-                level += 1
-                if lastnode and self.config.epub_tocdup:
```

Comment:
+1 for removing tocdup.
epub3 builder is experimental feature in 1.4.x. IMO, improvement is important than compatibility, so incompatibility is acceptable to me.